pkg/registry/core/interdomain/interdomain_nse_test.go

```diff
@@ -223,7 +223,7 @@ func TestInterdomainFloatingNetworkServiceEndpointRegistry(t *testing.T) {
 
 	domain3 := sandbox.NewBuilder(t).
 		SetNodesCount(0).
-		SetRegistrySupplier(func(context.Context, *url.URL, ...grpc.DialOption) registry2.Registry {
+		SetRegistrySupplier(func(context.Context, time.Duration, *url.URL, ...grpc.DialOption) registry2.Registry {
```

Comment:
Can we fix the artifact `registry2` ? :)

Reply:
Yep, fixed with `registryapi` for `github.com/networkservicemesh/api/pkg/api/registry` and no alias for `github.com/networkservicemesh/sdk/pkg/registry`.